compiler/src/parsing/parser_header.re

```diff
@@ -98,13 +94,26 @@ let mkid_expr = (loc, ns) =>
 
 let mkstr = (loc, s) => mkloc(s, to_loc(loc));
 
-let make_program = statements => {
+let make_module_alias = ident => {
+  switch (ident.txt) {
+  | IdentName(name) => name
+  | IdentExternal(_) =>
+    raise(
+      SyntaxError(
+        ident.loc,
+        "This module alias should be at most one level deep.",
```

Comment:
I think this error is fine, but here's a bikeshed:
```suggestion
        "A module alias cannot contain the `.` character.",
```
```suggestion
        "A module alias cannot contain the dot character.",
```

Reply:
Hmm, maybe. We might want to lean towards saying something along the lines of "you tried to use a name that normally references something within a module, but you're not allowed to do that here."

Reply:
Yeah, something like that.